<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveSatDialog</class>
 <widget class="QDialog" name="ActiveSatDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Active satellites</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>722</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QListWidget" name="satListWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QPushButton" name="applyButton">
        <property name="whatsThis">
         <string>Click to apply downlink frequency for the selected satellite.</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>423</width>
          <height>293</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>379</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Frequency [MHz]</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="freqCb">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Enter the selected satellite downlink frequency in MHz.</string>
             </property>
             <property name="whatsThis">
              <string>Enter the selected satellite downlink frequency in MHz.</string>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>1691.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1698.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1700.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1701.3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1702.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1707.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1700.4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8160.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8212.5</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="vhf_freqlabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>VHF freq</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QCheckBox" name="downconvertCb">
             <property name="text">
              <string>Downconvert</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1" colspan="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>158</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="toolTip">
              <string>http://www.oso.noaa.gov/poesstatus/</string>
             </property>
             <property name="whatsThis">
              <string>POES Satellite status web page.</string>
             </property>
             <property name="text">
              <string>&lt;a href=&quot;http://www.oso.noaa.gov/poesstatus/&quot;&gt;POES-Status&lt;/a&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <underline>false</underline>
              </font>
             </property>
             <property name="toolTip">
              <string>http://www.oso.noaa.gov/goesstatus/</string>
             </property>
             <property name="whatsThis">
              <string>GOES Satellite status web page.</string>
             </property>
             <property name="text">
              <string>&lt;a href=&quot;http://www.oso.noaa.gov/goesstatus/&quot;&gt;GOES-Status&lt;/a&gt;</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>RX Script</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>401</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="enableRXScriptCb">
             <property name="text">
              <string>Enable script</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QToolButton" name="rxscriptButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Arguments</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="4">
            <widget class="QTextEdit" name="rxscriptargEd">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QLineEdit" name="rxscriptEd"/>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="defaultrecScriptArgsBtn">
             <property name="text">
              <string>Default arguments</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="testrxscriptButton">
             <property name="text">
              <string>Test script</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Post processing script</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>401</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="enablePostProcScriptCb">
             <property name="text">
              <string>Enable script</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QToolButton" name="postrxscriptButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Arguments</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="4">
            <widget class="QTextEdit" name="postprocScriptArgEd">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QLineEdit" name="postprocScriptEd"/>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="testprocscriptButton">
             <property name="text">
              <string>Test script</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="defaultprocScriptArgsBtn">
             <property name="text">
              <string>Default arguments</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="testAllSatScriptButton">
      <property name="text">
       <string>Test all active satellite scripts</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActiveSatDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActiveSatDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
